File: TAF_test/Python/py_ui/schulte-playwright-ui-tests/logger/logger.py
import logging


class Logger:
    def __init__(self, name='schulte_playwright_tests'):
        self.logger = logging.getLogger(name)
        if not self.logger.handlers:
            self.logger.setLevel(logging.DEBUG)
            handler = logging.StreamHandler()
            handler.setLevel(logging.DEBUG)
            formatter = logging.Formatter(
                '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
            )
            handler.setFormatter(formatter)
            self.logger.addHandler(handler)

    def info(self, message):
        self.logger.info(message)

    def warning(self, message):
        self.logger.warning(message)

    def error(self, message):
        self.logger.error(message)

    def debug(self, message):
        self.logger.debug(message)

File: TAF_test/JS/cypress/cypress_malagahoy_web/cypress/support/Logger.js
class Logger {

    info(message) {
        cy.log(`INFO : ${message}`);
        console.log(`INFO : ${message}`);
    }

    warning(message) {
        cy.log(`WARNING : ${message}`);
        console.warn(message);
    }

    error(message) {
        cy.log(`ERROR : ${message}`);
        console.error(message);
    }

}

export default new Logger();